SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -281,17 +284,30 @@ bool FLocalDeploymentManager::TryStopLocalDeployment()
 	return bSuccess;
 }
 
-bool FLocalDeploymentManager::TryStartSpatialService()
+bool FLocalDeploymentManager::TryStartSpatialService(FString RuntimeIPToExpose)
 {
 	if (bSpatialServiceRunning)
 	{
 		UE_LOG(LogSpatialDeploymentManager, Verbose, TEXT("Tried to start spatial service but it is already running."));
 		return false;
 	}
+	else if (bStartingSpatialService)
+	{
+		UE_LOG(LogSpatialDeploymentManager, Verbose, TEXT("Tried to start spatial service but it is already being started."));
+		return false;
+	}
 
 	bStartingSpatialService = true;
 
 	FString SpatialServiceStartArgs = FString::Printf(TEXT("service start --version=%s"), *SpatialServiceVersion);
+
+	// Pass exposed runtime IP if one has been specified
+	if (RuntimeIPToExpose != TEXT("NONE"))
```

Comment:
Maybe it's some UE4 guideline I don't know of, but what is the reason for comparisons with TEXT("NONE") and not calling IsEmpty() on an empty string instead?

Reply:
I also think that the IsEmpty() check is cleaner, and "NONE" is not used anywhere else in our GDK. I've changed it now.